packages/hix/lib/Hix/Options.hs

```diff
@@ -257,11 +257,15 @@ ghcidParser cwd = do
   extra <- extraGhcidParser
   pure GhcidOptions {..}
 
+noInitGitAndFlakeParser :: Parser Bool
+noInitGitAndFlakeParser = switch (long "no-init-git-and-flake" <> short 'x' <> help "Skip git repo initialisation and nix flake lock")
```

Comment:
Maybe we can come up with a less specific and "sentencey" name – something like `--files-only` or `--generate-only`? Or even just `--files` or `--basic`? Even `--no-init` would seem sufficient to me. What do you think?

Reply:
I chose `--basic` as I think the other ones are a bit misleading and I couldn't come up with something better. And this way it can have a nice `short 'b'`which makes more sense. Let me know if it's okay with you.

Reply:
yep great!